metaflow/plugins/aip/aip.py

```diff
@@ -547,7 +547,10 @@ def _get_minutes_between_retries(node: DAGNode) -> Optional[str]:
         retry_deco = [deco for deco in node.decorators if deco.name == "retry"]
         if retry_deco:
             val = retry_deco[0].attributes.get("minutes_between_retries")
-            return f"{val}m" if isinstance(val, numbers.Number) else val
+            is_number = isinstance(val, numbers.Number) or (
+                isinstance(val, str) and val.isdecimal()
+            )
+            return f"{val}m" if is_number else val
```

Comment:
Hmm do we want to wait for 2 mins all the time? Just thinking how much time that'd potentially add to the integration test runs. Is there a way to be smarter here and use an exponential backoff?

Edit: To clarify I think this change given it's for the `_get_mins_between_retries` isn't the issue, this fix is good and should stay. But for the integration tests it'd be nice for those to run as quickly as possible, not sure what that means in terms of potential changes.

Reply:
Thanks for bringing this up!
I set `--with retry:minutes_between_retries=0` and `@retry(minutes_between_retries=0)` for integration tests!